integrations/docker/images/chip-build-nrf-platform/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG VERSION=latest
 FROM connectedhomeip/chip-build:${VERSION}
 
 # Compatible Nordic Connect SDK revision.
-ARG NCS_REVISION=v1.4.0
+ARG NCS_REVISION=v1.5.0
```

Comment:
It's already in another PR. Once that PR is merged you will also need to update .github/workflows to use the new image.